backend/typescript/migrations/2024.10.22T19.42.10.add-behaviour-columns.ts

```diff
@@ -0,0 +1,17 @@
+import { DataType } from "sequelize-typescript";
+import { Migration } from "../umzug";
+
+export const up: Migration = async ({ context: sequelize }) => {
+  await sequelize
+    .getQueryInterface()
+    .addColumn("behaviours", "parent_behaviour_id", {
+      type: DataType.INTEGER,
```

Comment:
you should have a foreign key here to itself. something like:

    pet_id: {
      type: DataType.INTEGER,
      allowNull: false,
      references: {
        model: "pets",
        key: "id",
      },
    },

(taken from backend/typescript/migrations/2024.07.06T15.03.01.create-pet-behaviour-table.ts)